database/index.js

```diff
@@ -14,43 +10,46 @@ const sequelize = new Sequelize('yelpPhotoGallery', 'root', '', {
   }
 });
 
-sequelize
-  .authenticate()
-  .then(() => {
-    console.log('Connection has been established successfully.');
-  })
-  .catch(err => {
-    console.error('Unable to connect to the database:', err);
-  });
-
-  const Users = sequelize.define('users', {
-    name: { type: Sequelize.STRING },
-    stars: { type: Sequelize.INTEGER },
-    reviews: { type: Sequelize.INTEGER }
-  });
-
-  const Businesses = sequelize.define('businesses', {
-    name: { type: Sequelize.STRING }
-  });
-
-  const Images = sequelize.define('images', {
-    image_url: { type: Sequelize.STRING },
-    helpfull_count: { type: Sequelize.INTEGER },
-    not_helpfull_Count: { type: Sequelize.INTEGER },
-    description: { type: Sequelize.STRING },
-    date: { type: Sequelize.DATEONLY },
-    business_id: { type: Sequelize.INTEGER}, //This needs to be foreign key refference
-    users_id: { type: Sequelize.INTEGER } //This needs to be foreign key refference
-  });
-
-
-  Users.sync({ force: true }).then(() => {
-    Users.bulkCreate(seed.seed.users, { logging: false })
-  });
-  Businesses.sync({ force: true }).then(() => {
-    Businesses.bulkCreate(seed.seed.businesses, { logging: false })
-  });
-  Images.sync({ force: true }).then(() => {
-    Images.bulkCreate(seed.seed.images,{ logging: false})
-  });
+const Users = sequelize.define('users', {
+  name: { type: Sequelize.STRING },
+  stars: { type: Sequelize.INTEGER },
+  reviews: { type: Sequelize.INTEGER }
+});
+
+const Businesses = sequelize.define('businesses', {
+  name: { type: Sequelize.STRING }
+});
+
+const Images = sequelize.define('images', {
+  imageUrl: { type: Sequelize.STRING },
+  helpfullCount: { type: Sequelize.INTEGER },
+  notHelpfullCount: { type: Sequelize.INTEGER },
+  description: { type: Sequelize.STRING },
+  date: { type: Sequelize.DATEONLY },
+  businessId: { type: Sequelize.INTEGER }, //This needs to be foreign key refference
+  usersId: { type: Sequelize.INTEGER } //This needs to be foreign key refference
+});
+
+
+
+module.exports.initialize = () => {
+  sequelize
+    .authenticate()
+    .then(() => {
+      console.log('Connection has been established successfully.');
+      return sequelize.sync({ force: false });
+    })
+    .then(() => {
+      console.log('Database tables have been synced');
+    })
+    .catch(err => {
+      console.error('Unable to connect to the database:', err);
+    });
+
+};
+
+module.exports.Users = Users;
+module.exports.Businesses = Businesses;
+module.exports.Images = Images;
```

Comment:
when declaring these exports, you could make this a little cleaner by utilizing es6 object property value shorthand and declare these in the same object all at once
```suggestion
module.exports = {
  Users,
  Businesses,
  Images,
};
```